### DISCUSSION TITLE
Am I missing something with the PowerBI report template downloads after 0.7 release?

### DISCUSSION THREAD

**Discussion by EQ-GALLI** [microsoft/finops-toolkit#1186] [Q&A] [answered]
I notice that whenever I try to download any template report on the Toolkit landing page, I am directed to the GitHub project, in the assets there are 2 reports both cost management report pbix files.  But all other templates are not available.

I have using a stored address in my browser history successfully pulled 0.6 versions down here is the address used: |
https://github.com/microsoft/finops-toolkit/releases/download/v0.6/CostSummary.pbix 

When I update the address to 0.7 I get the normal 404 page, but the download does not happen.

**Comment by lundejd** [comment]
I was looking into some of the .zip files on the .7 update details page and there is a PowerBI-demo.zip file that has what looks like the CostSummary.kql.pbix if you use Data Explorer, but is missing CostSummary.storage.pbix. There is a CostSummary.storage.pbit file elsewhere but that doesn't help.

I believe they forgot to include the hub storage CostSummary file which teaches me not to do these upgrades without checking all the files first because now I have a non-working CostSummary file.

**Comment by lundejd** [comment]
I just had a conversation with Michael Flanakin and was able to get what I needed from the .pbit file, so you may want to try that.

We have been customizing our .pbix files so every time a new release comes out I go in and copy/paste each function and paramater(if there are changes there) and of course the code on the front of CostDetails. By going into the .pbit file and instead of refreshing you just click the little (almost hidden) down arrow next to Load then select "edit" and you can now see all those bits.

**Comment by flanakin** [comment, marked as answer]
As of 0.7, there are 3 downloads for Power BI reports:
- PowerBI-demo.zip -- Demo reports with sample data (based on KQL reports)
- PowerBI-storage.zip -- Report templates that connect to storage
- PowerBI-kql.zip -- Report templates that connect to Data Explorer

Sorry for the confusion! We tried to update documentation to clarify this, but it was a bit of a miss in the  beginning. We haven't heard much since so hopefully that clarified the confusion 🤞

---

Inside scoop:
Previously, we had 5 PBIX files and 5 PBIT files that all connected to storage. This has always been a problem because they were generated manually from our source. This has always led to bugs here and there as manual steps are inadvertently missed. (Sorry! 😓) 

When we added Data Explorer support, we had to add 5 more of each. This would have made it difficult to find the reports you care about when you're scrolling thru a list of 20 files. This also would have increased the likelihood that we ended up with bugs from the manual release process. Since PBIX and PBIT are fundamentally the same, except for the pre-connected datasets with sample data, we kept one set of 5 for demo purposes and then kept two other sets for the storage and Data Explorer connectors. We used PBIT for those since they're smaller and we have a path to automation to reduce the likelihood of bugs going forward.

Of course, the renamed links resulted in their own bugs as we missed updating a few of the links in documentation and then some people struggled with the usability differences of PBIT files compared to PBIX files 🥴 We haven't heard much, but I don't know if that's because it's been 2 months or because we were just getting back the pain of changing. We'll keep monitoring and look for ways to optimize based on feedback. 

Thanks for your patience, support, and time! This is a labor of love, so those are all critical!
